### DISCUSSION TITLE
On Windows ?!?

### DISCUSSION THREAD

**Discussion by Giuliano3277** [1000001101000/Debian_on_Intel_Terastations#7] [Q&A]
Hello!
I have installed Windows 10 in this hardware. 
Can you have a suggestion to calm the main FAN ?

**Comment by 1000001101000** [comment]
Which model do you have?

Disclaimer: I have zero experience with controlling fans under windows.

For most of these devices the fan speed is controlled by the IT87 chip on the mainboard. A program that knows how to talk to the IT87 might be able to do what you want. I did a quick google search and “speedfan” looks like it could be worth a try.

Different boards use the IT87 differently so I doubt it will detect everything correctly automatically, it may show odd values for temps/voltages/speeds but it’s worth a try.

You could poke around at the firmware files buffalo provides for the device. I’d guess there IT87 driver and whatever program they use to control it is part of that package. I doubt it would work with windows 10 but it might provide some useful information.

**Comment by Giuliano3277** [comment]
The model is TS-QVL Series (TS-QVH4.0TL/R6EU).
Whit the software "Hardware Monitor" I see the Fan named "FANIN2" spinning at 2014-2020 RPM and this software is not able to modify this speed...
The software CPU-Z report LPCIO - ITE - IT8721

**Reply by 1000001101000** [reply]
Under Debian I can control it with PWM3 on my TS-QVL if that helps.
